docs/source/example_notebooks/gcm_linear_scm_bootstap_pvalues/test_false_discovery_rate_control.py

```diff
@@ -0,0 +1,77 @@
+#from dowhy.gcm.FalseDiscoveryRateControl import numberfy, adjusted_FDR_pvalues
+from false_discovery_rate_control import numberfy, adjusted_FDR_pvalues, adjusted_FDCR_pvalues
+import pytest
+import numpy as np
+import math
+from scipy.stats import rankdata
+
+def test_numberfy():
+    assert numberfy(17.1)   == 17.1
+    assert numberfy("3.7")  == 3.7
+    assert (math.isnan(numberfy("x.1")))
+    assert (math.isnan(numberfy(np.nan)))
+
+def test_adjFDR():
+    adjFDR=adjusted_FDR_pvalues((0.05, 0.01, 0.1))
+    assert abs(adjFDR[0] - 0.075)< 0.0001
+    assert abs(adjFDR[1] - 0.030)< 0.0001
+    assert abs(adjFDR[2] - 0.100)< 0.0001
+
+    adjFDR=adjusted_FDR_pvalues([0.01, "-", 0.1])
+    assert abs(adjFDR[0] - 0.020)< 0.0001
+    assert (math.isnan(adjFDR[1]))
+    assert abs(adjFDR[2] - 0.100)< 0.0001
+
+    adjFDR=adjusted_FDR_pvalues((0.01, 0.022, 0.023, 0.05,0.07,0.1))
+    assert abs(adjFDR[0] - 0.046)< 0.0001
+    assert abs(adjFDR[1] - 0.046)< 0.0001
+    assert abs(adjFDR[2] - 0.046)< 0.0001
+    assert abs(adjFDR[3] - 0.075)< 0.0001
+    assert abs(adjFDR[4] - 0.084)< 0.0001
+    assert abs(adjFDR[5] - 0.100)< 0.0001
+
+    adjFDR=adjusted_FDR_pvalues((0.01, np.nan , 0.023, "-",0.07,0.1))
+    # [0.04              nan 0.046             nan 0.09333333 0.1       ]
+    assert abs(adjFDR[0] - 0.040)< 0.0001
+    assert (math.isnan(adjFDR[1]))
+    assert abs(adjFDR[2] - 0.046)< 0.0001
+    assert (math.isnan(adjFDR[3]))
+    assert abs(adjFDR[4] - 0.09333333)< 0.00000001
+    assert abs(adjFDR[5] - 0.100)< 0.0001
+
+def test_adjusted_FDCR_pvalues():
```

Comment:
Can you add a test case where the adjustment of, e.g., `fdr_bh` would give false insights, but the `fdcr` does the correct thing?